lxd/cluster/heartbeat_test.go

```diff
@@ -142,9 +142,8 @@ func (f *heartbeatFixture) Grow() *cluster.Gateway {
 
 	state, gateway, address := f.node()
 	name := address
-
 	nodes, err := cluster.Accept(
-		targetState, target, name, address, cluster.SchemaVersion, len(version.APIExtensions))
+		targetState, target, name, address, cluster.SchemaVersion, len(version.APIExtensions), 1)
```

Comment:
Please use `osarch.ARCH_64BIT_INTEL_X86` instead of `1` and merge the two similar commits together.